datumaro/components/operations.py

```diff
@@ -915,6 +854,93 @@ def _check_groups_definition(self):
                     )
 
 
+@attrs
+class UnionMerge(Merger):
```

Comment:
Could we have a unit test for this?

Reply:
I created `test_can_merge_union` in test_ops.py. thank you!